CHANGELOG.md

```diff
@@ -28,6 +28,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - [mia-platform/#256](mia-platform/community/discussions/256) configurable collection tags.
 
+### Changed
+
+- [mia-platform/[#53](https://github.com/mia-platform/crud-service/issues/53) request to transition to a not allowed state returns 400 instead of 400
```

Comment:
Clearly a typo. Thanks for noticing.

```suggestion
- [mia-platform/[#53](https://github.com/mia-platform/crud-service/issues/53) request to transition to a not allowed state returns 400 instead of 404.
```